docs/sql-ref-syntax-aux-analyze-tables.md

```diff
@@ -0,0 +1,90 @@
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+The `ANALYZE TABLES` statement collects statistics about all the tables in a database to be used by the query optimizer to find a better query execution plan.
+
+### Syntax
+
+```sql
+ANALYZE TABLES [ { FROM | IN } database_name ] COMPUTE STATISTICS [ NOSCAN ]
+```
+
+### Parameters
+
+* **{ FROM `|` IN } database_name**
+
+    Specifies the name of the database to be analyzed. Without a database name, `ANALYZE` collects all tables in the current database that the current user has permission to analyze.
+
+* **[ NOSCAN ]**
+
+    Collects only the table's size in bytes ( which does not require scanning the entire table ).
+
+### Examples
+
+```sql
+CREATE DATABASE school_db;
+USE school_db;
+
+CREATE TABLE teachers (name STRING, teacher_id INT);
+INSERT INTO teachers VALUES ('Tom', 1), ('Jerry', 2);
+
+CREATE TABLE students (name STRING, student_id INT);
+INSERT INTO students VALUES ('Mark', 111111), ('John', 222222);
+
+ANALYZE TABLES IN school_db COMPUTE STATISTICS NOSCAN;
+
+SHOW TABLE EXTENDED IN school_db LIKE '*';
++------------+------------+--------------+----------------------------------------------------+
+|  database  | tableName  | isTemporary  |                    information                     |
++------------+------------+--------------+----------------------------------------------------+
+| school_db  | students   | false        | Database: school_db
+Table: students
+...
+Statistics: 24 bytes
+...
+
+| school_db  | teachers   | false        | Database: school_db
+Table: teachers
+...
+Statistics: 14 bytes
+...
++------------+------------+--------------+----------------------------------------------------+
+
+ANALYZE TABLES COMPUTE STATISTICS;
+
+SHOW TABLE EXTENDED IN school_db LIKE '*';
++------------+------------+--------------+----------------------------------------------------+
+|  database  | tableName  | isTemporary  |                    information                     |
++------------+------------+--------------+----------------------------------------------------+
+| school_db  | students   | false        | Database: school_db
+Table: students
+...
+Statistics: 24 bytes, 2 rows
+...
+| school_db  | teachers   | false        | Database: school_db
+Table: teachers
+...
+Statistics: 14 bytes, 2 rows
+...
++------------+------------+--------------+----------------------------------------------------+
+```
```

Comment:
Could you add a link to the `ANALYZE TABLE` page in the `Related Statements` section? Also, I think its better to add a link to this doc page in the `ANALYZE TABLE` page, too.